### PR TITLE
Add note on hostname getting updated when running KFP demo on a new cluster.

### PR DESCRIPTION
**Pull request by ashok133**
Added a note on hostname getting updated when running KFP demo on a new cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,11 @@ Running this demo uses the config file to generate a `trainer/` module that is
 compatible with CAIP. It also generates `orchestration/pipeline.py`, which
 compiles a Kubeflow Pipelines pipeline.
 
+> Note: If you're using a GKE cluster without Workload Identity configured, the
```
Comment (TheMichaelHu) [inline README.md]: I believe `>` is for quotes. Maybe use `_italics_` or `**emphasis**` instead?

I suggest changing the last sentence to
`If this occurs, you will need to update your config.yaml with the new Kubeflow Pipelines URL and rerun the demo.`

**Review by TheMichaelHu — CHANGES_REQUESTED** [review]

**Comment by TheMichaelHu** [comment]
It shouldn't matter, but you have a weird commit history in this branch. If you want a clean history, make sure you're pulling from master and rebasing.

File: README.md (hunk 1 above)
Comment (ashok133) [reply README.md]: Ack, done!

**Review by TheMichaelHu — APPROVED** [review]
LGTM

**Pull request merged**